tests/unit/customizations/s3/test_subcommands.py

```diff
@@ -103,9 +103,9 @@ def test_ls_command_for_bucket(self):
             .return_value.paginate
         # We should make no operation calls.
         self.assertEqual(call.call_count, 0)
```

Comment:
The line that creates the `call` object should probably be changed to:
```
call = self.session.create_client.return_value.list_objects_v2
```
instead of:
```
call = self.session.create_client.return_value.list_objects
```

Reply:
👍